docs/api/users/create.md

```diff
@@ -1,7 +1,136 @@
 ---
 id: create
-title: Create
+title: Create User
 sidebar_label: Create
+keywords:
+  - OpenHIM
+  - API
+  - User
+  - Create
+description: Create an OpenHIM User via the API
 ---
 
-Coming soon ...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Create User
+
+To create a user, you will need to make a TLS request to the OpenHIM API for the below method and endpoint and supply the JSON object for the user.
+
+
+```curl
+Method: POST
+Endpoint: {openhim_url}:8080/users
+Payload: JSON object of the user
+```
+
+## Example
+
+<Tabs
+  defaultValue="lang"
+  values={[
+    { label: 'Languages', value: 'lang' },
+    { label: 'Sample Data', value: 'sample' }
+  ]
+}>
+<TabItem value="lang">
+
+  Before we can send our request to the OpenHIM API we need to ensure that we construct our valid HTTP headers to successfully authenticate with the OpenHIM API.
+
+  <Tabs
+    defaultValue="nodejs"
+    values={[
+      { label: 'NodeJS', value: 'nodejs' },
+      { label: 'Bash', value: 'bash' }
+    ]
+  }>
+  <TabItem value="nodejs">
+
+  Copy the below code at the bottom of your nodejs script that handles the authentication of the OpenHIM headers as described in the [authentication section](../introduction/authentication.md). 
```

Comment:
This is from an old issue, but we need to update the link to the auth section as currently this is broken. We need to remove the `.md` from the link reference